index.js

```diff
@@ -55,12 +55,11 @@ module.exports = function(options){
 
             var addSourcemap = function(fileToAddTo, blessOutputIndex) {
                 if(!shouldCreateSourcemaps) return fileToAddTo;
-                applySourcemap(fileToAddTo, {
-                    version: 3,
-                    file: fileToAddTo.relative,
-                    sources: [file.relative],
-                    mappings: result.maps[blessOutputIndex]
-                });
+
+                var map = result.maps[blessOutputIndex];
```

Comment:
Actually the map created here will look different from the map previously, from what I noticed `file` and `sources` field looks different. I saw absolute path for `file` file for example.